freemocap/gui/qt/widgets/welcome_screen_dialog.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, gui_state: GuiState, kill_thread_event: threading.Event, pare
         multi_camera_docs_label.setWordWrap(True)
         self._layout.addWidget(multi_camera_docs_label, 1)
 
-        multi_camera_recording_doc_link_string = '&#10132; <a href="https://freemocap.github.io/documentation/getting_started/multi_camera_calibration/" style="color: #333333;"> Multi camera recording tutorial</a>'
+        multi_camera_recording_doc_link_string = '&#10132; <a href="https://freemocap.github.io/documentation/multi-camera-calibration.html</a>'
```

Comment:
I don't know where to see these links  (single/multi camera) in the gui?

I looked for them at roughly 0:32 in the associate PR video